components/wso2is.notification.event.handlers/pom.xml

```diff
@@ -44,13 +44,21 @@
                         </Import-Package>
                         <Export-Package>
                             org.wso2.is.notification.*;
+                            org.wso2.is.tenant.management.*;
                         </Export-Package>
                         <DynamicImport-Package>*</DynamicImport-Package>
                     </instructions>
                 </configuration>
             </plugin>
         </plugins>
     </build>
+    <properties>
+        <carbon.commons.version>4.10.19</carbon.commons.version>
+        <carbon.tenant.mgt.version>4.11.39</carbon.tenant.mgt.version>
+        <carbon.tenant.mgt.stub.version>4.9.34</carbon.tenant.mgt.stub.version>
+        <identity.organization.management.version>1.1.24</identity.organization.management.version>
+        <carbon.identity.version>7.8.23.33-SNAPSHOT</carbon.identity.version>
```

Comment:
lets use a already released is version

Reply:
this contains the config changes i added to identity framework.